src/cf.data.pre

```diff
@@ -1148,6 +1148,12 @@ DOC_START
 	to access some external data source.
 	Those which do are marked with the tag [slow], those which
 	don't are marked as [fast].
+	If Squid is reconfigured while evaluation of configuration directive ACLs
+	was suspended by a slow ACL, then, when resuming evaluation of ACLs for
+	that directive, Squid will continue to use directive ACLs as they were
+	configured before reconfiguration. After reconfiguration, Squid forgets an
+	"old" ACL when it completes the last evaluation of directive ACLs that
+	were using it. Squid uses reconfigured ACLs when starting new evaluations.
```

Comment:
I do not insist on adding this text, especially since this PR makes Squid ACL treatment more "natural" or "intuitive", but I think we should document what happens to slow ACLs being evaluated during reconfiguration. It is difficult (for me) to do so well in the confinements of a cf.data.pre paragraph, but I think it is still better than being silent about this behavior. We can put a more verbose text (with examples and such) in the FAQ [section](https://wiki.squid-cache.org/SquidFaq/SquidAcl) already referenced below.

Reply:
From the questions and confusion I see on the mailing lists, I believe the new behaviour is one most if not all admin already expect to be happening.

IMO the text should probably be written a lot simpler. For example; the second sentence does not seem to contain information pertinent to admin uses. Also, try reducing the number of commas in the first sentence to make it clear and easily understood.